bluetooth/src/commonMain/kotlin/device/DeviceState.kt

```diff
@@ -230,7 +443,7 @@ sealed class ConnectableDeviceStateImpl {
             deviceConnectionManager.readRssi()
         }
 
-        suspend fun requestMtu(mtu: Int): Boolean {
+        suspend fun requestMtu(mtu: MTU): Boolean {
```

Comment:
With fresh look at, I think we can add RequestMTU, Pair, UnPair into DeviceAction, so we can handle callbacks in similar ways we do for write/read, what do you think?

Reply:
Not a bad idea, although, this would mean you cannot do a read characteristic while requesting MTU. So Im not willing to do that rewrite right in this PR.

Reply:
It can be different queue of actions, of course this if out of scope of this PR